app/code/Magento/Quote/Model/GuestCart/GuestCouponManagement.php

```diff
@@ -8,7 +8,6 @@
 
 use Magento\Quote\Api\GuestCouponManagementInterface;
 use Magento\Quote\Api\CouponManagementInterface;
-use Magento\Quote\Model\QuoteIdMask;
```

Comment:
We can't remove this line because `Magento\Quote\Model\QuoteIdMask` used in `get` method without the fully qualified name